lib/src/generated/INetworkListManagerEvents.dart

```diff
@@ -0,0 +1,66 @@
+// INetworkListManagerEvents.dart
+
+// THIS FILE IS GENERATED AUTOMATICALLY AND SHOULD NOT BE EDITED DIRECTLY.
+
+// ignore_for_file: unused_import
+
+import 'dart:ffi';
+
+import 'package:ffi/ffi.dart';
+
+import '../com/combase.dart';
+import '../constants.dart';
+import '../exceptions.dart';
+import '../macros.dart';
+import '../ole32.dart';
+import '../structs.dart';
+import '../utils.dart';
+
+import 'IUnknown.dart';
+
+/// @nodoc
+const CLSID_NetworkListManagerEvents = '';
+
+/// @nodoc
+const IID_INetworkListManagerEvents = '{DCB00001-570F-4A9B-8D69-199FDBA5723B}';
```

Comment:
@timsneath the metadata DO find a GUID for `INetworkListManagerEvents`. But I don't know how to write the right example. The usage could be found https://github.com/fluttercommunity/plus_plugins/blob/79d62dd32579855d8293bf6f07fe4487c1ad5e45/packages/connectivity_plus_windows/windows/network_manager.cpp#L19-L62

```cpp
class NetworkListener final : public INetworkListManagerEvents {
  // ...
}

bool NetworkManager::StartListen(NetworkCallback pCallback) {
  // ...
        pListener = new NetworkListener(pCallback);
        hr = pConnectPoint->Advise((IUnknown *)pListener, &dwCookie);
  // ...
}
```

Reply:
Right -- `INetworkListManagerEvents` has a GUID, but there's no class `NetworkListManagerEvents` (as far as I can figure out). Likely there's another class that implements `INetworkListManagerEvents`; if so, then you should have what you need with my change, because you can call `QueryInterface` on that to get the right interface.